grouper/src/grouper/edu/internet2/middleware/grouper/plugins/FrameworkStarter.java

```diff
@@ -12,26 +14,47 @@
 import org.osgi.framework.launch.Framework;
 import org.osgi.framework.launch.FrameworkFactory;
 
+import java.io.FileOutputStream;
 import java.io.IOException;
+import java.io.OutputStream;
+import java.io.StringReader;
 import java.lang.reflect.InvocationTargetException;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.Certificate;
+import java.security.cert.CertificateException;
+import java.security.cert.CertificateFactory;
+import java.util.Arrays;
 import java.util.Dictionary;
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.Hashtable;
 import java.util.Map;
 import java.util.ServiceLoader;
 import java.util.Set;
+import java.util.regex.Pattern;
 
 /**
  * Class used to start up the OSGI framework
  *
  * @author jj
  */
 public class FrameworkStarter {
+    private final static Log LOG = GrouperUtil.getLog(FrameworkStarter.class);
+
     private final static FrameworkStarter frameworkStarter = new FrameworkStarter();
 
+    // properties
+    public final static String GROUPER_OSGI_ENABLE = "grouper.osgi.enable";
```

Comment:
do we continue to use `osgi` namespace, or do we use more generic `plugins`?

Reply:
I would vote to use osgi

Reply:
after our previous conversation, should likely be the opposite. since the grouper project has no intention of implementing osgi, some other term should be used. this statement is true for all questions related to osgi, something else should be used and documented.